packages/grafana-toolkit/docker/grafana-plugin-ci-e2e/scripts/deploy.sh

```diff
@@ -56,11 +56,15 @@ mv $HOME/go/bin/mage /usr/local/bin
 /bin/rm -rf /tmp/mage
 /bin/rm -rf $HOME/go
 
+#
```

Comment:
Why not just remove it instead of commenting it out? I don't see how keeping the commented out code will be helpful.

Reply:
agree, meant to make this draft so i could clean it up 👍